build.js

```diff
@@ -35,18 +36,32 @@ var isOptional = function isOptional(category) {
   return (category || '').indexOf('annex b')>-1 || category === 'pre-strawman' || category === 'strawman (stage 0)';
 };
 
+var byTestSuite = function(suite) {
+  return function(browser) {
+    return Array.isArray(browser.test_suites) ? browser.test_suites.indexOf(suite)>-1 : true;
```

Comment:
Oh right sorry, duh. The lack of spaces around the operator threw me :-) I'll clean that up later, the spaces should be there.